### PR TITLE
Fix vars_prompt "private" description and improve example

### LINKED ISSUE #44906: vars_prompt docs does not say default is private
##### SUMMARY

https://docs.ansible.com/ansible/2.6/user_guide/playbooks_prompts.html

Does not mention that prompts are private by default.  The "alternative form" section is probably superfluous.

##### ISSUE TYPE
 - Documentation Report

##### COMPONENT NAME

vars_prompt

##### ANSIBLE VERSION

```
2.6
```

### PR DESCRIPTION
**Pull request by zharalim**
##### SUMMARY

Remove example about "private: no" and make the first example show both the default case and the "private: no" case.

Fixes #44906

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
vars_prompt

##### ANSIBLE VERSION
N/A

##### ADDITIONAL INFORMATION
N/A

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_prompts.rst [hunk 1]
```diff
@@ -12,27 +12,29 @@ in a push-script.
 
 Here is a most basic example::
 
-    ---
-    - hosts: all
-      remote_user: root
+      ---
```
Comment (jborean93) [inline docs/docsite/rst/user_guide/playbooks_prompts.rst]: Why the extra 2 spaces, having 4 spaces as the indent is enough for RST to put this in a codeblock.

Comment (zharalim) [reply docs/docsite/rst/user_guide/playbooks_prompts.rst]: Fixed

**Review by jborean93 — APPROVED** [review]
Looks good to me, will let @acozine have a review.

**Review by samccann — APPROVED** [review]

**Comment by acozine** [comment]
Nice PR! Fixed a problem and shortened the docs at the same time! Thanks @zharalim.

**Comment by zharalim** [comment]
Thanks! No problem!

**Pull request merged**